### ISSUE TITLE
Alow setting nudge size

### ISSUE THREAD

**Issue by gabrielvaz** [littlebusters/Adjust-size-by-shortcut-for-AdobeXD#1]
Please, let us configurate the size of the nudge. Instead of 10px I'd like to switch to 8px for example

**Comment by craigmdennis** [comment]
If you open the plugins folder, you can open `main.js` and change the `10`s to `8`. That's what I've done.

**Comment by craigmdennis** [comment]
@gabrielvaz I made my own and also included the ability to _nudge_ by 8px and settings to choose https://github.com/craigmdennis/adobe-xd-nudge-and-resize/releases/

If you set the shortcuts correctly then it overrides XDs `10px` default if you're on a mac. I've no idea about settings shortcuts on Windows machines.

I've submitted it to Adobe but if you want it _now_ then use the link above and download the `.xdx` file.
